dolphin/drivers/api.py

```diff
@@ -58,11 +58,13 @@ def remove_storage(self, context, storage_id):
 
     def get_storage(self, context, storage_id):
         """Get storage device information from storage system"""
-        pass
+        driver = self.driver_manager.get_driver(context, storage_id)
```

Comment:
did you test it, get_driver doen't has the parameter storage_id

Reply:
'get_driver' requires storage_id as argument.